src/testers.jl

```diff
@@ -89,6 +91,7 @@ end
    - If `check_inferred=true`, then the inferrability (type-stability) of the `frule` is checked,
      as long as `f` is itself inferrable.
    - `fkwargs` are passed to `f` as keyword arguments.
+   - `testset_name`: if provided, the name of the testset used to wrap the tests.
```

Comment:
```suggestion
- `testset_name`: if provided, the name of the testset used to wrap the tests.
  Otherwise it is determined from the function and argument types.
```